scripts/e2e-experimental.sh

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env bash
+set -xeuf
+
+# This is the full workflow test of the ilab CLI.
+#
+# We expect it to be run anywhere `ilab` would run, including the instructlab
+# container images.
+#
+# It represents the workflow a typical user would run through with ilab.
+
+# generic globals
+BOLD='\033[1m'
+NC='\033[0m' # No Color
+PRESERVE=0
+
+# path and token globals
+SCRIPTDIR=$(dirname "$0")
+E2E_TEST_DIR=""
+CONFIG_HOME=""
+DATA_HOME=""
+CACHE_HOME=""
+CONFIG_HOME=""
+HF_TOKEN=${HF_TOKEN:-}
+
+# model globals
+GRANITE_EMBEDDING_MODEL="ibm-granite/granite-embedding-125m-english"
+GRANITE_7B_MODEL="instructlab/granite-7b-lab"
+
+# t-shirt size globals
+DEV=0
+TECH=0
+CUSTOM=0
+
+check_flags() {
+    if [ "${DEV}" -ne 1 ] && [ "${TECH}" -ne 1 ] && [ "${CUSTOM}" -ne 1 ]]; then
+         echo "ERROR: Must specify a feature scope flag when invoking this script."
+         usage
+         exit 1
+    fi
+}
+
+########################################
+# This function initializes the directory
+# used during end-to-end testing. This function
+# also creates the directory structure which
+# instructlab expects to see at the time of execution.
+# Arguments:
+#   None
+# Globals:
+#   DATA_HOME
+#   CACHE_HOME
+#   CONFIG_HOME
+#   E2E_TEST_DIR
+#   HOME
+# Outputs:
+#   None
+########################################
+
+init_e2e_tests() {
```

Comment:
shouldn't we also init the `ILAB_FEATURE_SCOPE` env var introduced in the feature-gates PR?